.github/workflows/spack_build.yml

```diff
@@ -130,8 +130,11 @@ jobs:
           spack:
             specs:
             - ${{ matrix.spack_spec }} target=x86_64_v2
+            - openblas
+            - libevent~openssl
             concretizer:
-              reuse: dependencies
+              reuse: true
+              unify: true
```

Comment:
@cameronrutherford how did you figure out these?

Reply:
reuse true is just a sane default. It makes sure you always pull a version of a package from a build cache when possible. There are different options as detailed [here](https://spack.readthedocs.io/en/latest/build_settings.html).

I was having trouble with the blas provider specification, so I decided to put it in the core spack spec. Then, by specifying unify to be true, I was able to ensure the version of hiop was also built with the version of blas that I wanted.